Fix bugs removing geometry

mwoehlke-kitware opened the pull request:
Fix tests for looking up a geometry ID. If the look-up fails, the variable will be `undefined`, but the test was also failing if the ID is `0`, which is a valid ID. Fix type getting the geometry from the frame data.

matthewma7 commented on web_external/util/annotationGeometryParser.js:
```diff
@@ -155,7 +155,7 @@ class AnnotationGeometryContainer {
+        if (geomId !== undefined) {
             // Geometry already exists for the specified state; look it up and
             // modify it in place
             let geomToChange = this._frameMap(frame).get(trackId);
```
I think this _framemap access is wrong

mwoehlke-kitware replied:
In what way?

matthewma7 replied:
Isn't _framemap is a map, so it needs **.get()** instead of **()** to get value?

mwoehlke-kitware replied:
Ah, yes. (I'm sure there is a technical term for the brain's tendency to fail to notice a word that is expected to be present, but isn't, but I can't find it... anyway, fixed; thanks!)

mwoehlke-kitware committed:
Fix bugs removing geometry

Fix tests for looking up a geometry ID. If the look-up fails, the
variable will be undefined, but the test was also failing if the ID is
0, which is a valid ID. Fix type getting the geometry from the frame
data.

matthewma7 reviewed (approved):
Looks good.
